bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HPlusHF/HPlusCharm_4FS_MuRFScaleDynX0p50_HToGG_M125_TuneCP5_13p6TeV_amcatnloFXFX_pythia8/HPlusCharm_4FS_MuRFScaleDynX0p50_HToGG_M125_TuneCP5_13p6TeV_amcatnloFXFX_pythia8_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model loop_sm_MSbar_yb_yc-yc4FS
+
+define p = g u u~ d d~ s s~ c c~
+define j = g u u~ d d~ s s~ c c~
+
+generate p p > h [QCD] @0
+add process p p > h j [QCD] @1
```

Comment:
where do you define the H->gg decay, the name is "HToGG"

Reply:
The H->gg decay is set in the pythia fragment

Reply:
then you need to remove the "HtoGG" in the name